galley/pkg/config/resource/stringmap.go

```diff
@@ -18,21 +18,21 @@ package resource
 type StringMap map[string]string
 
 // Clone the StringMap
-func (s *StringMap) Clone() StringMap {
-	if *s == nil {
+func (s StringMap) Clone() StringMap {
```

Comment:
does this mean we now copy the whole string map?

Reply:
maps aren't copied in golang ... they're always by reference